CMakeLists.txt

```diff
@@ -229,9 +229,20 @@ elseif (${TARGET_PLATFORM} STREQUAL "android")
     )
 endif()
 
+if (WIN32)
+    set(PLATFORM_FILES
+        ${PLATFORM_FILES}
+        ${PROJECT_SOURCE_DIR}/src/platform/windows/backtrace_windows.c
```

Comment:
Nitpick: `windows/backtrace_windows.c` (and `posix/backtrace_posix.c`) seems redundant. I think `windows/backtrace.c` and `posix/backtrace.c` is enough.

Reply:
Done